sbncode/FlashMatch/FlashPredict.hh

```diff
@@ -148,87 +292,74 @@ private:
   const double fBeamWindowStart, fBeamWindowEnd;
   const double fFlashStart, fFlashEnd;
   const unsigned fTimeBins;
-  const bool fSelectNeutrino, fOnlyPrimaries;
+  const bool fSelectNeutrino;
+  const bool fOnlyCollectionWires;
+  const bool fForceConcurrence;
   const bool fUseUncoatedPMT, fUseOppVolMetric;//, fUseCalo;
   const bool fUseARAPUCAS;
+  const bool fStoreCheatMCT0;
   const std::string fInputFilename;
   const bool fNoAvailableMetrics, fMakeTree;
   const double fChargeToNPhotonsShower, fChargeToNPhotonsTrack;
-  const double fMinHitQ, fMinSliceQ, fQScale;
-  const double fMinOpHPE, fMinFlashPE, fPEScale;
-  std::string fDetector; // SBND or ICARUS
-  bool fSBND, fICARUS;
-  std::unique_ptr<opdet::PDMapAlg> fPDMapAlgPtr;
+  const double fMinHitQ, fMinSpacePointQ, fMinParticleQ, fMinSliceQ;
+  const unsigned fMaxFlashes;
+  const double fMinOpHPE, fMinFlashPE;
+  const art::ServiceHandle<geo::Geometry> fGeometry;
+  const std::string fDetector; // SBND or ICARUS
+  const bool fSBND, fICARUS;
+  const std::unique_ptr<opdet::PDMapAlg> fPDMapAlgPtr;
+  const size_t fNTPC;
   const int fCryostat;  // =0 or =1 to match ICARUS reco chain selection
   // geo::CryostatID fCryostat;  // TODO: use this type instead
-  std::unique_ptr<geo::CryostatGeo> fGeoCryo;
-  const int fNBins;
+  const std::unique_ptr<geo::CryostatGeo> fGeoCryo;
+  const std::list<double> fWiresX_gl;
   const double fDriftDistance;
-  size_t fNTPC;
+  const int fXBins;
+  const double fXBinWidth;
+  const std::string fRR_TF1_fit, fRatio_TF1_fit;
+  unsigned fYBins,fZBins;
+  double fYLow, fYHigh, fZLow, fZHigh;
+  double fYSkewThreshold, fZSkewThreshold;
+  const double fYBiasSlope, fZBiasSlope;
   unsigned fDriftVolumes;
   unsigned fTPCPerDriftVolume;
   const unsigned fOpDetNormalizer;
   const double fTermThreshold;
-  std::list<double> fWiresX_gl;
-  // std::vector<double> fPMTChannelCorrection;
 
-  unsigned fPeakCounter = 0;
-  std::vector<recob::OpHit>::iterator fOpH_beg, fOpH_end;
-  const art::ServiceHandle<geo::Geometry> geometry;
+  const unsigned kRght = 0;
```

Comment:
If these are known at compile time, is there a reason not to use a `constexpr`

Reply:
Good idea.